### ISSUE TITLE
Dimension of evaluation

### ISSUE THREAD

**Issue by sm176357** [DrSleep/tensorflow-deeplab-resnet#185]
Is the evaluation done on the complete image size or is the evaluation done on the downsized image ?
For example, if an image is of size (320,240) and the feature map size leading to the softmax is (160,120) or (40,30), is the evaluation done on (320,240) or on the (40,30) vs gt_resized(40,30) ?

**Comment by DrSleep** [comment]
the evaluation is done on the original size - (320, 240) in your example.

**Comment by sm176357** [comment]
Thanks for the clarification.
